src/navigation/wallet/screens/AddWallet.tsx

```diff
@@ -498,10 +498,24 @@ const AddWallet: React.FC<AddWalletScreenProps> = ({navigation, route}) => {
       const wallet = await _addWallet(_associatedWallet, walletName);
 
       if (!withinReceiveSettings) {
-        navigation.navigate('WalletDetails', {
-          walletId: wallet.id,
-          key,
-          skipInitializeHistory: false, // new wallet might have transactions
+        navigation.reset({
```

Comment:
apparently navigation.reset(...) is deprecated in the latest versions of react native. You should use CommonaAtions.reset(...) instead. Here is an example:
```navigation.dispatch(
                CommonActions.reset({
                  index: 1,
                  routes: [
                    {
                      name: 'Tabs',
                      params: {screen: 'Home'},
                    },
                    {
                      name: 'Wallet',
                      params: {screen: 'KeyOverview', params: {id: key.id}},
                    },
                  ],
                }),
              );```

Reply:
+1 Thanks!

Reply:
It doesn't work. Maybe commonActions requires more params.

Reply:
Just added navigation.dispatch 👍🏼 
Thanks!